src/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.csproj

```diff
@@ -15,7 +15,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Content Include="$(RepoRoot)eng\pkg\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
-    <Content Include="$(RepoRoot)eng\pkg\_._" Pack="true" PackagePath="lib\netstandard2.0\" />
```

Comment:
What's the story with this change? It is pretty hidden with a lot of formatting changes. If we really want to make this change, we should make it separate from the rest of these changes.

Reply:
This is actually a warning you pointed out to me a while ago. We are including the `_._` file in the package, which is supposed to mean the folder is empty, but the folder isn't really empty. So it throws a build warning. I am happy to move it to a new PR though.